tests/unit/preprocessor/_time/test_time.py

```diff
@@ -1022,6 +1022,16 @@ def test_standardized_anomalies(period, standardize=True):
             )
 
 
+@pytest.mark.parametrize('period, reference', PARAMETERS)
+def test_anomalies_preserve_metadata(period, reference, standardize=False):
+    cube = make_map_data(number_years=2)
+    cube.var_name = "si"
+    cube.units = "m"
+    result = anomalies(cube, period, reference, standardize=standardize)
+    assert result.var_name == cube.var_name
+    assert result.units == cube.units
```

Comment:
Perhaps we can use
```suggestion
    assert result.xml() == cube.xml()
```
to verify that *all* metadata matches. That seems to pass for my use case, and it's more complete. For example, one of the subsequent problems I got with this bugs is that the time bounds were removed. These are included in the xml.

Reply:
that'll not work, there are differences in the xml's that got nothing to do with the actual metadata; I have enhanced the test to look for identical bounds and identical coord objects